diff --git a/node_modules/react-navigation-stack/src/utils/validateDeprecatedConfig.tsx b/node_modules/react-navigation-stack/src/utils/validateDeprecatedConfig.tsx
index f97a7e4..f298cad 100644
--- a/node_modules/react-navigation-stack/src/utils/validateDeprecatedConfig.tsx
+++ b/node_modules/react-navigation-stack/src/utils/validateDeprecatedConfig.tsx
@@ -1,5 +1,5 @@
 import { StyleProp, ViewStyle } from 'react-native';
-import { StackNavigationOptions, StackNavigationConfig } from '../vendor/types';
+import { StackNavigationConfig, StackNavigationOptions } from '../vendor/types';
 
 type Validation = {
   message: string;
@@ -38,7 +38,6 @@ const validations: Record<string, Validation> = {
   ...[
     'cardShadowEnabled',
     'cardOverlayEnabled',
-    'cardStyle',
     'headerBackTitleVisible',
     'onTransitionStart',
     'onTransitionEnd',
diff --git a/node_modules/react-navigation-stack/src/vendor/views/Stack/Card.tsx b/node_modules/react-navigation-stack/src/vendor/views/Stack/Card.tsx
index 3ea8548..150c5ff 100755
--- a/node_modules/react-navigation-stack/src/vendor/views/Stack/Card.tsx
+++ b/node_modules/react-navigation-stack/src/vendor/views/Stack/Card.tsx
@@ -1,32 +1,21 @@
+import Color from 'color';
 import * as React from 'react';
 import {
-  Animated,
-  View,
-  StyleSheet,
-  ViewProps,
-  StyleProp,
-  ViewStyle,
-  Platform,
-  InteractionManager,
+  Animated, InteractionManager, Platform, StyleProp, StyleSheet, View, ViewProps, ViewStyle
 } from 'react-native';
 import {
-  PanGestureHandler,
-  State as GestureState,
-  PanGestureHandlerGestureEvent,
+  PanGestureHandler, PanGestureHandlerGestureEvent, State as GestureState
 } from 'react-native-gesture-handler';
 import { EdgeInsets } from 'react-native-safe-area-context';
-import Color from 'color';
-import StackGestureRefContext from '../../utils/GestureHandlerRefContext';
+import {
+  GestureDirection,
+  Layout, StackCardStyleInterpolator, TransitionSpec
+} from '../../types';
 import CardAnimationContext from '../../utils/CardAnimationContext';
+import StackGestureRefContext from '../../utils/GestureHandlerRefContext';
 import getDistanceForDirection from '../../utils/getDistanceForDirection';
 import getInvertedMultiplier from '../../utils/getInvertedMultiplier';
 import memoize from '../../utils/memoize';
-import {
-  TransitionSpec,
-  StackCardStyleInterpolator,
-  GestureDirection,
-  Layout,
-} from '../../types';
 
 type Props = ViewProps & {
   index: number;
@@ -485,7 +474,7 @@ export default class Card extends React.Component<Props> {
                   pointerEvents="none"
                 />
               ) : null}
-              <View ref={this.content} style={[styles.content, contentStyle] as any}>
+              <View ref={this.content} style={[styles.content, contentStyle,{backgroundColor:"red"}] as any}>
                 <StackGestureRefContext.Provider value={this.gestureRef}>
                   <CardAnimationContext.Provider value={animationContext}>
                     {children}
diff --git a/node_modules/react-navigation-stack/src/vendor/views/Stack/CardContainer.tsx b/node_modules/react-navigation-stack/src/vendor/views/Stack/CardContainer.tsx
index 2ca12de..1211095 100644
--- a/node_modules/react-navigation-stack/src/vendor/views/Stack/CardContainer.tsx
+++ b/node_modules/react-navigation-stack/src/vendor/views/Stack/CardContainer.tsx
@@ -1,17 +1,15 @@
 import * as React from 'react';
-import { Animated, View, StyleSheet, StyleProp, ViewStyle } from 'react-native';
+import { Animated, StyleProp, StyleSheet, View, ViewStyle } from 'react-native';
 import { NavigationState as StackNavigationState } from 'react-navigation';
-import { Props as HeaderContainerProps } from '../Header/HeaderContainer';
-import Card from './Card';
-import HeaderHeightContext from '../../utils/HeaderHeightContext';
 import useTheme from '../../../utils/useTheme';
 import {
-  Route,
-  Scene,
-  Layout,
-  StackHeaderMode,
-  TransitionPreset,
+  Layout, Route,
+  Scene, StackHeaderMode,
+  TransitionPreset
 } from '../../types';
+import HeaderHeightContext from '../../utils/HeaderHeightContext';
+import { Props as HeaderContainerProps } from '../Header/HeaderContainer';
+import Card from './Card';
 
 type Props = TransitionPreset & {
   index: number;
